### PR TITLE
Fix default folder not showing in light theme after apply customization.

### LINKED ISSUE #852: Default folder not showing in light theme after applying customization
- [x] I'm sure this issue is not a *duplicate*?

The default folder is not showing in light themes when the user selects `Apply Icons Customization`.

This is a bug I introduced when I increased the code coverage by tests. I distinctly remember that I wasn't sure how to cover the `toggle folders` presets.

### PR DESCRIPTION
**Pull request by JimiC**
***Fixes #852***

**Changes proposed:**
* [x] Fix

**Things I've done:**
* [x] My pull request fixes an issue, I referenced the issue.

### CODE REVIEW

**Comment by JimiC** [comment]
@robertohuertasm I strongly suggest a fix release with this.

**Comment by JimiC** [comment]
Damn. `-4` lines coverage. I was afraid of that.

File: src/icon-manifest/manifestMerger.ts [hunk 1]
```diff
@@ -8,18 +8,18 @@ export function mergeConfig(
   supportedFolders: models.IFolderCollection,
   iconGenerator: models.IIconGenerator): models.IIconSchema {
 
-  const dFiles = customFiles ? customFiles.default : null;
-  const dFolders = customFolders ? customFolders.default : null;
-  const sFiles = customFiles ? customFiles.supported : null;
-  const sFolders = customFolders ? customFolders.supported : null;
+  const defaultCustomFiles = customFiles ? customFiles.default : null;
```
Comment (JimiC) [inline src/icon-manifest/manifestMerger.ts]: @robertohuertasm I changed the names so they are more clear on what they reference.

**Comment by robertohuertasm** [comment]
It seems that bithound complains about `tslint` not being upgraded

**Comment by JimiC** [comment]
Yeah, we already discussed that. For this PR `bitHound` reports an error because it's dumb. It can't distinguish that I have an account on it and tries to analyze on the commit.

**Comment by robertohuertasm** [comment]
I'm not sure to understand this. Shouldn't we update to 5.0.0 to be ok with it?

**Comment by JimiC** [comment]
https://github.com/palantir/tslint/issues/2485

**Comment by robertohuertasm** [comment]
But I'm referring to `bitHound - Dependencies`, which states that our `tslint` version is outdated.

**Comment by JimiC** [comment]
OK. Let's make this clearer. Updating to `tslint 5.0.0` introduces an issue which will force us to disable the `no-unused-expression` rule. Until the `tslint` team fixes this potential bug we stick to `4.5.1` no matter what `bitHound` says.

**Comment by robertohuertasm** [comment]
Ok. Now I understand. Sorry but I didn't know about this bug and was completely unaware of prior discussions. 👍

**Comment by JimiC** [comment]
Technically it was an implicit discussion. I assumed that you could refer to my reasoning from the link issue. Obviously, I assumed wrong.

**Comment by robertohuertasm** [comment]
I missed the version on top of the Github issue... thought you were talking about the `4.5.1`... 🙄

**Pull request merged**